include/usermode.h

```diff
@@ -53,6 +53,12 @@ extern void __user_text printf(const char *fmt, ...);
 extern void *__user_text mmap(void *va, unsigned long order);
 extern void __user_text munmap(void *va, unsigned long order);
 
+extern long __user_text syscall(long syscall_nr, long arg1, long arg2, long arg3,
+                                       long arg4, long arg5);
+
+extern long __user_text sysenter(long syscall_nr, long arg1, long arg2, long arg3,
+                                        long arg4, long arg5);
+
```

Comment:
See https://github.com/KernelTestFramework/ktf/pull/290/commits/2d5d2d59729b0ab0db0bd02a0b82a323c3b2d587#r1307195325, this low level API shouldn't be exported, imho, but a _mode switching API_ instead.